src/utils/number.tsx

```diff
@@ -50,3 +51,16 @@ export const formatCurrency = (balance: BigNumber): string => {
     };
     return balance.toFormat(2, BigNumber.ROUND_DOWN, fmt);
 };
+
+export const numDigits = (value: BigNumber): number => {
+    return value.toString().length;
+};
+
+export const roundToNearestInteger = (
```

Comment:
Hard for me to understand what the math is doing below, so just wanted to confirm, is this rounding to the nearest integer, as in if it is .5 or above it goes to 1, and less than .5 it goes to 0?